webapp/app/forms/flows/step_chooser.py

```diff
@@ -38,6 +38,12 @@ def _get_possible_redirect(self, step_name, stored_data):
                 return dbg[0].name
             else:
                 return self.first_step.name
+        elif step_name == 'start_next':
+            dbg = self.default_data()
+            if dbg:
+                return dbg[0].name
+            else:
+                return self.determine_next_step(self.first_step.name, {}).name
```

Comment:
I am wondering if we could put this into the eligibility step chooser instead of having that here for all step choosers. The reason is that this is very specific for the eligibility flow that the first step is not the actual first step :) 

Furthermore, we should add tests for that 🙈

Reply:
Done :)